docs/Files/UI/Solution-Manager/Library/MasterJobs/Copy-Jobs.md

```diff
@@ -0,0 +1,9 @@
+# Copying Jobs
+
+To copy a Master Job, go to **Library** > **Master Jobs** and select **Copy**.
+
```

Comment:
I would add a step telling the user they must select one job before copy is enabled